lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.spec.ts

```diff
@@ -694,43 +665,32 @@ describe('StartProcessCloudComponent', () => {
         });
 
         it('should call service to start process if required fields provided', () => {
-            component.currentCreatedProcess = fakeProcessInstance;
+            component.processForm.controls['processInstanceName'].setValue('My Process 1');
```

Comment:
I think we should have just a test from the UI where the user provides the Process instance name, the process definition key and when clicking on submit we can spy on the startProcessSpy but making sure the payload is the expected not just toHaveBeenCalled

Reply:
Done, test in: "should call service with the correct parameters when button is clicked"